client/dfclient/include/menu_state.hpp

```diff
@@ -16,7 +20,20 @@ struct MenuState
 	StateType Update(Messages) override;
 
 private:
-	bool TryConnect();
+	void TryConnect();
+	void ProcessMatchmakerData(std::string data);
+	void ShowMessage(std::string message);
+	void MatchmakerLayout();
+	void DirectConnectLayout();
+
+	ClientMode clientMode = ClientMode::Matchmaker;
+	std::string matchmakerAddress = "localhost";
+	std::string matchmakerPort = "8000";
+	std::string directAddress = "localhost";
+	std::string directPort = "63987";
+
+	char buf0[64];
+	char buf1[64];
```

Comment:
Is there no way to use less non-descriptive names? :D They seem to be indentical in functionality and one of them  is always a nickname, while second is __sometimes__ a server adress. Maybe they could be assigned a single role and named as such.

Reply:
Renamed buffers and assigned roles to them